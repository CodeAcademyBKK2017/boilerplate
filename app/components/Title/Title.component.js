import React, {Component} from 'react';
import styles from './Title.style.js';
import {
  Text
  , TextInput
  , View
} from 'react-native';

class Title extends Component {
  render () {
    return (
      <View>
        <Text style={styles.text}>Note Title</Text>
        <TextInput
          style={styles.textinput}
        />
      </View>
    );
  }
}

export default Title;