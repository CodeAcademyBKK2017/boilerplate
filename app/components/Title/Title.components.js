import React, {Component} from 'react';
import styles from './Title.style';
import {
  Text,
  TextInput,
  View
} from 'react-native';

class Title extends Component {
  render () {
    return (
      <View style={styles.title}>
        <View style={styles.textRow}>
          <View style={styles.titleLeft}>
            <Text style={styles.titleText}>Note Title</Text>
          </View>
    
          <View >
            <Text style={styles.titleEn}>en</Text>
          </View>
        </View>
        
        <TextInput 
          style={styles.titleInput}
          placeholder='Tasks for today'
        />
       
      </View>
    );
  }
}

export default Title;