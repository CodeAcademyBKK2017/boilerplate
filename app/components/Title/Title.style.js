import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  text: {
    fontWeight: 'bold'
  },
  textinput: {
    minWidth: '100%',
    borderColor: 'gray',
    borderWidth: 1
  }
});