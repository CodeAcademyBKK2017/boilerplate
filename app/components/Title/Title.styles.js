import React, {StyleSheet} from 'react-native';

export default StyleSheet.create({
  container: {
    width: '100%',
    paddingBottom: 10
  },
  title: {
    fontSize: 13,
    color: '#000',
    fontWeight: 'bold'
  },
  textInput: {
    marginTop: 10,
    marginBottom: 10,
    borderWidth: 1,
    borderStyle: 'solid',
    borderColor: '#c4c4c5',
    borderRadius: 3,
    paddingTop: 12,
    paddingBottom: 12,
    paddingLeft: 12,
    paddingRight: 12,
    backgroundColor: '#ffffff'
  }
});