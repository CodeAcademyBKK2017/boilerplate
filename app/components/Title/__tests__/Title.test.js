
import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import Title from '../Title.component';

describe('Title', () => {
  it('renders correctly', () => {
    const snapshot = renderer.create(<Title />);
    expect(snapshot).toMatchSnapshot();
  });
});