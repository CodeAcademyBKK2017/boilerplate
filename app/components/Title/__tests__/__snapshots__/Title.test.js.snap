// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    Note Title
  </Text>
  <TextInput
    allowFontScaling={true}
    style={
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "minWidth": "100%",
      }
    }
  />
</View>
`;
